components/About/AboutPage/index.tsx

```diff
@@ -14,80 +14,83 @@ import { styles } from './styles'
 
 const Map = dynamic(() => import('../Map'), { ssr: false })
 
-const AboutPage = (props: { theme: Object }) => (
-  <div css={styles(props.theme)}>
-    <Head>
-      <script
-        async
-        defer
-        src='https://api.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js'
-      />
-      <link
-        href='https://api.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.css'
-        rel='stylesheet'
-      />
-    </Head>
+const AboutPage = () => {
+  const theme = useTheme()
+  return (
+    <div css={styles(theme)}>
+      <Head>
+        <script
+          async
+          defer
+          src='https://api.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js'
+        />
+        <link
+          href='https://api.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.css'
+          rel='stylesheet'
+        />
+      </Head>
 
-    <Title typed>./about</Title>
-    <div className='relative-container py-5'>
-      <Container
-        id='_what_we_do'
-        className='py-5 bg-secondary rounded-0 d-md-flex'
-      >
-        <Row className='d-flex align-items-center'>
-          <Col xs={12} lg={5}>
-            <h2 className='header'>We build software for charities</h2>
-            <p className='lead m-lg-0'>
-              Coders for Causes is a not for profit organisation that empowers
-              charities and other not for profit organisations by connecting
-              them with university students to develop technical solutions. We
-              are a student-run club based in Perth, Western Australia with a
-              wide range of clients. Whether you are looking for technical
-              advice or a long term project, get in touch with us for more
-              information.
-            </p>
+      <Title typed>./about</Title>
+      <div className='relative-container py-5'>
+        <Container
+          id='_what_we_do'
+          className='py-5 bg-secondary rounded-0 d-md-flex'
+        >
+          <Row className='d-flex align-items-center'>
+            <Col xs={12} lg={6}>
+              <h2 className='header'>We build software for charities</h2>
+              <p className='lead m-lg-0'>
+                Coders for Causes is a not for profit organisation that empowers
+                charities and other not for profit organisations by connecting
+                them with university students to develop technical solutions. We
+                are a student-run club based in Perth, Western Australia with a
+                wide range of clients. Whether you are looking for technical
+                advice or a long term project, get in touch with us for more
+                information.
+              </p>
+            </Col>
+          </Row>
+        </Container>
+        <div className='map'>
```

Comment:
@JeremiahPinto I moved the map element outside the container so the container would not get stretched when applying width changes to the map. This fixed many issues.